pkg/lib/cockpit-dark-theme.ts

```diff
@@ -33,7 +33,7 @@ function changeDarkThemeClass(documentElement, dark_mode) {
     }
 }
 
-function _setDarkMode(_style) {
+function _setDarkMode(_style?: string) {
     const style = _style || localStorage.getItem('shell:style') || 'auto';
```

Comment:
Could this become non-optional and instead get a default `auto` value?

Reply:
Not from my understanding of how this works.  The very next line queries the default from `localStorage()` before falling back to `auto`.